medical-clinic/src/main/resources/liquibase-changelog/versions/0006_MEDICAL_CLINIC.yaml

```diff
@@ -0,0 +1,23 @@
+databaseChangeLog:
+  - changeSet:
+      id: 6
+      author: Mateusz
+      changes:
+        - addColumn:
+            tableName: PATIENT
+            columns:
+              - column:
+                  name: user_id
+                  type: int
+                  constraints:
+                    foreignKey: true
+                    foreignKeyName: user_id
+        - addColumn:
+            tableName: DOCTOR
+            columns:
+              - column:
+                  name: user_id
+                  type: int
+                  constraints:
+                    foreignKey: true
+                    foreignKeyName: user_id
```

Comment:
czemu nie ma reference do ktorej kolumny z ktorej tabeli referuje to foreign key?